src/figuresClasses.ts

```diff
@@ -1,11 +1,103 @@
-export interface Figure {}
+export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: 'triangle' | 'circle' | 'rectangle';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  color: 'red' | 'green' | 'blue';
+
+  shape: 'triangle';
```

Comment:
The `shape` property should be declared as `readonly` to match the `Figure` interface.